test_data/controller/jxsv.sdp

```diff
@@ -0,0 +1,12 @@
+v=0
+o=- 1543226715 1543226715 IN IP4 {{ src_ip }}
+s=Demo Video Stream
+t=0 0
+m=video {{ dst_port }} RTP/AVP 97
+c=IN IP4 {{ dst_ip }}/32
+b=AS:{{ bit_rate }}
+a=source-filter: incl IN IP4 {{ dst_ip }} {{ src_ip }}
+a=ts-refclk:ptp=IEEE1588-2008:EC-46-70-FF-FE-00-CE-DE:0
+a=rtpmap:96 {{ media_type }}/90000
+a=fmtp:96 packetmode=0; profile={{ profile }}; level={{ level }}; sublevel={{ sublevel }}; depth={{ depth }}; width={{ width }}; height={{ height }}; exactframerate={{ exactframerate }}; {{ "interlace; " if interlace else "" }}sampling={{ sampling }}; colorimetry={{ colorimetry }}; TCS={{ transfer_characteristic }}; {{ "RANGE=FULL; " if fullrange else "" }}SSN=ST2110-22:2022; TP={{ st2110_21_sender_type }}
```

Comment:
I undid @David-P-B's introduction of a config SDP preference for the `SSN`, because it applies to video/raw and video/jxsv and indeed video/smpte291, but the allowed values are different.

Theoretically receivers could reject SDP with a value they don't recognise,  so we may need to parameterize it somehow...

Reply:
OK - shall we think on that for inclusion after this merge?

Reply:
yep, I'll just make sure everything is consistent for now 👍